doc/styles/TextBox.md

```diff
@@ -7,83 +7,83 @@ uid: Uno.Themes.Styles.TextBox
+| `OutlinedTextBoxPlaceholderForegroundDisabled`      | `SolidColorBrush` | `OnSurfaceLowBrush`           |
+| `OutlinedTextBoxHeaderForeground`                   | `SolidColorBrush` | `OnSurfaceVariantBrush`       |
+| `OutlinedTextBoxHeaderForegroundPointerOver`        | `SolidColorBrush` | `OnSurfaceVariantBrush`       |
+| `OutlinedTextBoxHeaderForegroundFocused`            | `SolidColorBrush` | `PrimaryBrush`                |
+| `OutlinedTextBoxHeaderForegroundDisabled`           | `SolidColorBrush` | `OnSurfaceLowBrush`           |
+| `OutlinedTextBoxFontFamily`                         | `FontFamily`      | `MaterialMediumFontFamily`    |
+| `OutlinedTextBoxFontWeight`                         | `String`          | `BodyLargeFontWeight`         |
+| `OutlinedTextBoxFontSize`                           | `Double`          | `BodyLargeFontSize`           |
+| `OutlinedTextBoxCharacterSpacing`                   | `Int32`           | `BodyLargeCharacterSpacing`   |
+| `FilledTextBoxFontFamily`                           | `FontFamily`      | `MaterialMediumFontFamily`    |
+| `FilledTextBoxFontWeight`                           | `String`          | `BodyLargeFontWeight`         |
+| `FilledTextBoxFontSize`                             | `Double`          | `BodyLargeFontSize`           |
+| `FilledTextBoxCharacterSpacing`                     | `Int32`           | `BodyLargeCharacterSpacing`   |
+| `FilledTextBoxBorderThicknessNormal`                | `Double`          | `TextBoxOutlinedStrokeHeight` |
+| `FilledTextBoxBorderThicknessFocused`               | `Double`          | `TextBoxFocusStrokeWidth`     |
+| `MaterialTextBoxClearGlyphWidth`                    | `Double`          | 20                            |
+| `MaterialTextBoxClearGlyphHeight`                   | `Double`          | 20                            |
+| `MaterialFilledTextBoxCornerRadius`                 | `CornerRadius`    | 4,4,0,0                       |
+| `MaterialFilledTextBoxPadding`                      | `Thickness`       | 16,7                          |
+| `MaterialFilledTextBoxMinHeight`                    | `Double`          | 58                            |
+| `MaterialFilledTextBoxBorderHeightFocused`          | `Double`          | 2                             |
+| `MaterialOutlinedTextBoxBorderThickness`            | `Double`          | 1                             |
+| `MaterialOutlinedTextBoxCornerRadius`               | `CornerRadius`    | 4                             |
+| `MaterialOutlinedTextBoxPadding`                    | `Thickness`       | 8                             |
+| `MaterialOutlinedTextBoxMinHeight`                  | `Double`          | 56                            |
+| `MaterialOutlinedTextBoxBorderThicknessFocused`     | `Double`          | 2                             |
+| `MaterialOutlinedTextBoxBorderThicknessPointerOver` | `Double`          | 2                             |
```

Comment:
@Kunal22shah, @Xiaoy312 I imagined that with this other PR (https://github.com/unoplatform/Uno.Themes/pull/1491#discussion_r1815570378) we will align the "Material" starting term here ?

Reply:
im just recopying the template, dont really care being correct/wrong here
it should not be fixed here anyways